ui/components/app/alert-system/alert-modal/alert-modal.tsx

```diff
@@ -345,6 +350,9 @@ export function AlertModal({
           startAccessory={headerStartAccessory}
           paddingBottom={0}
           display={headerStartAccessory ? Display.InlineFlex : Display.Block}
+          closeButtonProps={{
+            'data-testid': 'alert-modal-close-button',
+          }}
```

Comment:
Fixes type error

### Before
![Screenshot 2024-08-20 at 8 56 19 AM](https://github.com/user-attachments/assets/467f1f5d-aaf2-4d30-8a24-d69a6b891e52)

### After

![Screenshot 2024-08-20 at 8 56 58 AM](https://github.com/user-attachments/assets/5f9d8315-d6fa-4fa7-9ead-9075a7612d58)

Reply:
Thanks for pointing it out and fixing it 🙌